errors/errors.go

```diff
@@ -10,3 +10,7 @@ func CheckError(err error) {
 		log.Fatal(err)
 	}
 }
+
+func CheckError2(_ interface{}, err error) {
```

Comment:
Can we avoid this `CheckError2` name please? I cannot propose a better name, so instead can you move it to `fixture.go` and rename to `checkError` ?

Reply:
It's a rubbish name. I've been trying to think of something better, but nothing has come to mind.  So I'll do this.

Reply:
How about `FailOnErr`?

Reply:
Sounds good!